index.html

```diff
@@ -22,7 +22,7 @@
         border: solid 1px silver;
         background-color: whitesmoke;
         padding: 15px;
-      }
+      } 
       #title,
       #description {
         text-align: center;

```

Comment:
It's preferred to use tags instead of Id of elements for creating CSS rules. for example, `h1` is more desirable than `#title`.